src/Microsoft.Extensions.ApiDescription.Client/build/Microsoft.Extensions.ApiDescription.Client.targets

```diff
@@ -279,8 +295,17 @@
       unique.
     -->
     <ItemGroup>
-      <Compile Remove="@(ServiceFileReference -> '%(OutputPath)')" />
-      <Compile Include="@(ServiceFileReference -> '%(OutputPath)')" SourceDocument="%(FullPath)" />
+      <TypeScriptCompile Remove="@(ServiceFileReference -> '%(RelativeOutputPath)')" />
+      <TypeScriptCompile Include="@(ServiceFileReference -> '%(RelativeOutputPath)')"
+          Condition="'%(ServiceFileReference.Extension)' == '.ts'">
+        <SourceDocument>%(ServiceFileReference.FullPath)</SourceDocument>
+      </TypeScriptCompile>
+
+      <Compile Remove="@(ServiceFileReference -> '%(RelativeOutputPath)')" />
```

Comment:
Should the condition from the `Include` apply here? We don't want to accidentally remove things we wouldn't add back